DiscordPy/bot.py

```diff
@@ -70,11 +72,12 @@ async def getHelp(ctx, *arg):
 
 @bot.command(name='meme')
 async def getMeme(ctx, *arg):
-    if (len(imageList) == 0):
-        hot_python = subreddit.new(limit=20)
+    delta_time = datetime.datetime.now() - prev_time

```

Comment:
I need to change variable 'prev_time' in this function

Reply:
We can find a better name for prev_time, perhaps something that shows "Hey, we're grabbing the current clock at the time of the bot being called."

Reply:
What about changing 'prev_time' to 'prev_call_meme_time', I'm not too good with my variable naming